README.md

```diff
@@ -1,117 +1,108 @@
+**A library for secure smart contract development** written in Cairo for [Starknet](https://starkware.co/product/starknet/), a decentralized ZK Rollup.
 
-## Usage
-
-> ## ⚠️ WARNING! ⚠️
->
+> **Warning**
 > This repo contains highly experimental code.
-> Expect rapid iteration.
+> It has no code coverage checks.
+> It hasn't been audited.
 > **Use at your own risk.**
 
-### First time?
+## Usage
 
-Before installing Cairo on your machine, you need to install `gmp`:
+> **Warning**
+> Expect rapid iteration.
+> Some contracts or features are not ready to be deployed.
+> Check the **Unsupported** section below.
 
-```bash
-sudo apt install -y libgmp3-dev # linux
-brew install gmp # mac
-```
+### Prepare the environment
 
-> If you have any troubles installing gmp on your Apple M1 computer, [here’s a list of potential solutions](https://github.com/OpenZeppelin/nile/issues/22).
+1. [Install cairo](https://book.cairo-lang.org/ch01-01-installation.html)
+2. [Install scarb](https://docs.swmansion.com/scarb/download)
 
 ### Set up your project
 
-Create a directory for your project, then `cd` into it and create a Python virtual environment.
+Create a new project and `cd` into it.
 
 ```bash
-mkdir my-project
-cd my-project
-python3 -m venv env
-source env/bin/activate
+scarb new my_project && cd my_project
 ```
 
-Install the [Nile](https://github.com/OpenZeppelin/nile) development environment and then run `init` to kickstart a new project. Nile will create the project directory structure and install [the Cairo language](https://www.cairo-lang.org/docs/quickstart.html), a [local network](https://github.com/Shard-Labs/starknet-devnet/), and a [testing framework](https://docs.pytest.org/en/6.2.x/).
+It should look like this
 
 ```bash
-pip install cairo-nile
-nile init
-```
-
-### Install the library
+$ ls
 
-```bash
-pip install openzeppelin-cairo-contracts
+Scarb.toml src
 ```
 
-> ⚠️ Warning! ⚠️  
-Installing directly the `main` branch may contain incomplete or breaking implementations, download [official releases](https://github.com/OpenZeppelin/cairo-contracts/releases/) only.
-
-### Use a basic preset
+### Install the library
 
-Presets are ready-to-use contracts that you can deploy right away. They also serve as examples of how to use library modules. [Read more about presets](https://docs.openzeppelin.com/contracts-cairo/0.6.1/extensibility#presets).
+Edit `scarb.toml` and add:
 
-```cairo
-// contracts/MyToken.cairo
-
-%lang starknet
-
-from openzeppelin.token.erc20.presets.ERC20 import (
-    constructor,
-    name,
-    symbol,
-    totalSupply,
-    decimals,
-    balanceOf,
-    allowance,
-    transfer,
-    transferFrom,
-    approve,
-    increaseAllowance,
-    decreaseAllowance
-)
+```toml
+[dependencies]
+openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.7.0" }
 ```
 
-Compile and deploy it right away:
+Build the project to download it:
```

Comment:
```suggestion
Build the project to download it.
```

Reply:
From my understanding, I think we should be using `:` when the idea is not yet completed (to be a sentence), like "The contents of `my_project` should look like this:", because "this" is not clear yet. `.` should be used in the rest of the cases like this one, where the sentence (idea) is complete.

Reply:
`:` is also to show something or, like this case, to give an example

Reply:
i think using both `.` and `:` is acceptable, we don't need to stick with a single one as long as usage makes sense.